scripts/linux/appimage/soh.sh

```diff
@@ -5,14 +5,18 @@ export PATH="$HERE"/bin:"$HERE"/usr/bin:"$PATH"
 export LD_LIBRARY_PATH="$HERE"/usr/lib:"$LD_LIBRARY_PATH"
 export ZENITY=$(command -v zenity)
 
-while [[ ! -e "$PWD"/oot.otr ]]; do
+if [ -z ${SHIP_HOME+x} ]; then
+export SHIP_HOME=.
```

Comment:
This should be an absolute path. When unset, it wont extract the assets from the ROM, and if a valid oot.otr exists, it will attempt to write logs to the read-only portion of the appimage.

![image](https://user-images.githubusercontent.com/62252937/185040232-514fe8b6-24d1-425a-8259-32567a6c1941.png)

This causes the error message to be less helpful, and trigger incorrect hash error.

`export SHIP_HOME="$PWD"` resolves the errors.